db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/Scheduler.java

```diff
@@ -26,348 +26,400 @@
 import java.time.Duration;
 import java.time.Instant;
 import java.util.*;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
+import java.util.concurrent.*;
```

Comment:
Yeah I see I have a bit of a whitespace issue in the project. I will fix that afterwards, in another PR, to make it more consistent.

Reply:
(the placement of this comment was probably better before I disabled whitespace changes... )